m4/lua.m4

```diff
@@ -34,7 +34,7 @@ dnl
 AC_DEFUN([KYUA_LUA], [
     lua_found=no
 
-    for lua_release in 5.2 5.1; do
+    for lua_release in ${LUA_VERSION:-5.4 5.3 5.2 5.1}; do
```

Comment:
A subtle difference:
```suggestion
    for lua_release in ${LUA_VERSION} 5.4 5.3 5.2 5.1; do
```
If LUA_VERSION is defined and everything is found for LUA_VERSION, then use it. Otherwise, use the hardcoded versions in order.

Reply:
We want it to error if the specified `LUA_VERSION` is not found instead of using other version that happens to be installed.

Reply:
Ah, that’s fair..